packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -37,6 +40,28 @@ class FloatingActionButton extends StatefulComponent {
 }
 
 class _FloatingActionButtonState extends State<FloatingActionButton> {
+  Performance _childSegue = new Performance(duration: _kFloatingActionButtonChildSegue);
+  Widget _child;
+
+  void initState() {
+    super.initState();
+    _child = new RotationTransition(
```

Comment:
Why not create this object in the build function like we do normally?  There shouldn't be any benefit to caching the widget.